test/fixture_test.js

```diff
@@ -1827,6 +1827,26 @@ asyncTest('should allow FormData as data type (#133)', function() {
 	xhr.send(data);
 });
 
+asyncTest('fixture returns the old fixture callback when fixtures are removed (#34)', function() {
+	var funcA = function(){
+		return "foo";
+	};
+	fixture("/services/thing", funcA);
+  
+	// in a test, remove default fixture and provide your own
+	var oldFixtures = fixture("/services/thing", null);
+	fixture("/services/thing", function(){});
+	QUnit.deepEqual(oldFixtures, [{fixture: funcA, url: '/services/thing'}]);
+	fixture(oldFixtures);
+
+	var xhr = new XMLHttpRequest();
+	xhr.addEventListener('load', function() {
+		start();
+	});
+	xhr.open('GET', '/services/thing');
+	xhr.send();
```

Comment:
Is the XHR needed? If so, I don’t understand what’s going on here and we should add some more comments to make it clear.

Reply:
@chasenlehara can you please explain what is not clear? I just hit the fixture under the test with XHR request.

Reply:
Here’s my understanding of what’s happening in this test:

1. Setting up a fixture
2. Removing the fixture
3. Defining a new fixture
4. Testing that the old fixture was returned when it was removed
5. Setting up the old fixture again
6. Making an XHR that will call the old fixture

I’m not sure why steps 3, 5, and 6 happen, because only 1, 2, and 4 seem necessary for the test.

Reply:
@chasenlehara Thank you, I changed it.
It was just to make sure the returned fixture really works as expected.